packages/indexer/src/plugins/persistence.test.ts

```diff
@@ -105,47 +104,42 @@ describe("Persistence", () => {
     });
 
     // Delete value
-    await store.del();
+    store.del();
 
     // Check there's no data
-    latest = await store.get();
+    latest = store.get();
     expect(latest.cursor).toBeUndefined();
     expect(latest.filter).toBeUndefined();
 
-    await db.close();
+    db.close();
   });
 
   it("should work with indexer and store cursor of last message", async () => {
     const client = new MockClient<MockFilter, MockBlock>((request, options) => {
       return messages;
     });
 
-    const persistence = sqlitePersistence<MockFilter, MockBlock, MockRet>({
-      driver: sqlite3.Database,
-      filename: "file:memdb1?mode=memory&cache=shared",
-    });
+    const db = new Database("file:memdb1?mode=memory&cache=shared");
```

Comment:
Can we make this one (and the others) `:memory:` too? or it won't work for the test?